bin/dev

```diff
@@ -1,9 +1,3 @@
 #!/usr/bin/env sh
 
-if ! command -v foreman &> /dev/null
-then
-  echo "Installing foreman..."
-  gem install foreman
-fi
-
-foreman start -f Procfile
+bundle exec puma -C config/puma.rb
```

Comment:
since we aren't using Heroku, I'm thinking we go with @aaronbrezel 's suggestion and nix the `foreman` installation just to run a `Procfile` when we could just move the contents of the `Procfile` here — what do we think?

Reply:
to document, since I didn't know this: it looks like [Procfile's are used by Heroku deployments](https://www.codementor.io/@populardemand/the-heroku-procfile-1sxnqu1rqo). This seems fine _for our own use_, along with not using `foreman`.

BUT If we eventually want to merge some of this back to the main Klaxon branch though, I'd recommend not doing this, since it would remove some of the necessary infrastructre if someone was using Heroku

Reply:
Just to underline the link between `foreman` and `Procfile`s, `foreman`'s [documentation](https://www.rubydoc.info/gems/foreman/0.87.2) states that the gem is specifically for managing `Procfile`-based applications. 

I think nixing it is probably okay! One thing I would like to know more about before providing a strong opinion is our ultimate goal for this repo and our adoption of klaxon in general. 

Is our goal to get a klaxon server running on an ecs cluster s.t. it's available for journalists in our newsroom?

Reply:
If we are going with the containerized approach, we could cut out `bin/dev` and the `Procfile` out of the equation entirely and run `bundle exec` directly through whatever docker startup setup we use

Reply:
I assumed that we wouldn't want to maintain code that is exclusively for Heroku to make our own navigation of the app easier. I _think_ we are able to cherrypick changes to open a PR with against the original Klaxon just to help with Docker development. But, this is fine if we want to prioritize making contributions easy.